src/app/home-page/home-page.component.ts

```diff
@@ -13,18 +14,22 @@ export class HomePageComponent implements OnInit {
   public campaigns: CampaignSummary[];
   public loading = false; // Server render gets initial result set; set true when filters change.
   public resetSubject: Subject<void> = new Subject<void>();
+  public videoEmbedUrl: SafeResourceUrl;
+  public youtubeId = '5RWDzzBg-go';
 
   private perPage = 6;
   private query: {[key: string]: any};
 
   constructor(
     private campaignService: CampaignService,
+    private sanitizer: DomSanitizer,
   ) {
   }
 
   ngOnInit() {
     this.setDefaults();
     this.run();
+    this.videoEmbedUrl = this.sanitizer.bypassSecurityTrustResourceUrl(`https://www.youtube.com/embed/${this.youtubeId}`);
```

Comment:
As this one's not dynamic I wonder if we should just hardcode it in the template to avoid the weight of all this?